pyproject.toml

```diff
@@ -9,7 +9,8 @@ requires-python = ">=3.8"
 dependencies = [
   "h5py>=3.0",  # requires numpy
   "numpy",
-  "PyYAML>=5.1"
+  "PyYAML>=5.1",
+  "tqdm"
```

Comment:
Is the extra dependency necessary here? I guess you might have added it since the filtering takes some time?

Reply:
I can remove if desired, just wanted something to show that things are happening while you wait